app/request.go

```diff
@@ -60,14 +85,108 @@ func NewRequestCommand(getRequestClientFn GetRequestClientFn) (CommandOut, error
 			Flags: []cli.Flag{
 				&cli.StringFlag{
 					Name:     "request-id",
-					Usage:    "The request-id of the asynchronous request",
+					Usage:    "The request-id of the request",
```

Comment:
`The request's ID` could be less circular, or `Unique identifier of the request`

Reply:
actually we should probably change this to operation id. to keep things in consistent with the new apis
will address this in the next PR

Reply:
That makes sense